src/ClientConfig.cc

```diff
@@ -44,13 +44,6 @@ class gz::fuel_tools::ClientConfigPrivate
                 homePath, ".gz", "fuel");
 
             this->servers.push_back(ServerConfig());
-            // Add in fuel.ignitionrobotics.org as another default server
```

Comment:
Have we resolved the login issue on the Fuel server? If not, I'd rather not make this change since we don't have a way to fix the models there, especially if some of them end up in the Harmonic demo.

Reply:
I think the login issue fix is being rolled out now. I can log back into fuel.